schema/config_blobset_blob.json

```diff
@@ -11,13 +11,20 @@
       "type": "string"
     },
     "base64": {
-      "type": "string"
+      "type": "string",
+      "pattern": "^[0-9A-Za-z=+/]{4,}$"
     },
     "url": {
-      "type": "string"
+      "type": "string",
+      "format": "uri"
     },
     "sha256": {
-      "type": "string"
+      "type": "string",
+      "pattern": "^[0-9A-Za-z]{64}$"
```

Comment:
what do you think about just making it all-lower or all-upper? Reason being, then it's easier to do a naive "string compare" to see if "hash matched" -- otherwise it takes extra code that somebody is likely to get wrong...  I'd rather force the generation to be all-lower than rely on the equality check.

Reply:
I'm fine with all lowercase because that's customary. I was wondering if device makers would have other opinions, especially from non-Unix platforms.

Reply:
Also this is a mistake and it should be a-f not a-z.